providers/teamleader/handlers.go

```diff
@@ -0,0 +1,67 @@
+package teamleader
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/naming"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+)
+
+const (
+	objectNameSuffix = ".list"
```

Comment:
would be more readable if we call this something like `apiListSuffix` (can be anything else too, just that it needs to communicate that this is specific to listing, not the object itself)